### ISSUE TITLE
build for mac os x arm64 version 5.47.0 wasn't created/uploaded to conda-forge

### ISSUE THREAD

**Issue by unhyperbolic** [conda-forge/maxima-feedstock#35]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

I just tried: conda install maxima==5.47.0

and got the following:

Retrieving notices: ...working... done
Channels:
 - conda-forge
 - defaults
Platform: osx-arm64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - maxima==5.47.0

Current channels:

  - https://conda.anaconda.org/conda-forge
  - defaults

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

### Installed packages

```shell
Python 3.11.0
```

### Environment info

```shell
Python 3.11.0
```

**Comment by saraedum** [comment]
This was explicitly by @isuruf in cfd419f8e5639015bd916fe30c541b49729f734a.

**Comment by saraedum** [comment]
i.e., in this PR https://github.com/conda-forge/maxima-feedstock/pull/27.

It doesn't say why. But I guess it simply doesn't build?

**Comment by GZTimeWalker** [comment]
I've noticed this problem for a long time, my sage environment can't be upgraded because it depends on this package.

```
Could not solve for environment specs
The following package could not be installed
└─ sage 10.3**  is not installable because it requires
   └─ maxima >=5.46.0,<6 , which does not exist (perhaps a missing channel).
```

**Comment by saraedum** [comment]
@GZTimeWalker you should be able to use the non-arm version of sage as a work around.

**Comment by GZTimeWalker** [comment]
> @GZTimeWalker you should be able to use the non-arm version of sage as a work around.

Yes, so I've been waiting for this package upgrade for a long time, and now I stumble upon this issue.

**Comment by isuruf** [comment]
cross-compiling doesn't work. Need to build it manually

**Comment by isuruf** [comment]
@saraedum I uploaded a package at https://anaconda.org/isuruf/maxima/files. Here's the log. https://gist.github.com/isuruf/1dbd364b29f74976cb567e9cd704ee3c

@ocefpaf, mind approving the manual build upload?

**Comment by ocefpaf** [comment]
> @ocefpaf, mind approving the manual build upload?

Sure. I have to confess I forgot the process but if all you need is a +1, let's do it.

I believe all core members have a token, if you do now I can upload it for you.

**Comment by isuruf** [comment]
Yeah, just a +1 after inspecting the logs. I do have a token and can upload.

**Comment by saraedum** [comment]
@isuruf, can this be closed?

**Comment by GZTimeWalker** [comment]
Hey @isuruf ! can you do another manual build for the latest depends?

latest package `_3`: `__osx >=10.13, ecl >=24.5.10,<24.5.11.0a0, gmp >=6.3.0,<7.0a0, readline >=8.2,<9.0a0`
the last arm64 package `_2`: `ecl >=23.9.9,<23.9.10.0a0, gmp >=6.3.0,<7.0a0, readline >=8.2,<9.0a0`

we stuck on the `ecl` version requirement.
